subsys/usb/class/Kconfig

```diff
@@ -107,6 +107,21 @@ config BLUETOOTH_BULK_EP_MPS
 	help
 	  Bluetooth device class bulk endpoint size
 
+config USB_DEVICE_BT_H4
+	bool "USB Bluetooth H4 Device Class Driver"
+	select BT
+	select BT_HCI_RAW
+	help
+	  USB Bluetooth H4 device class driver
+
+config BT_H4_BULK_EP_MPS
+	int
+	depends on USB_DEVICE_BT_H4
+	default 64
+	range 8 64
```

Comment:
This `range 8 64` is probably not necessary, is there any benefit to have smaller endpoint buffers?

Reply:
Nope, actually this might not work anymore since the buffer must be big enough to be used with a command.

Reply:
Shouldn't we then make it hard coded? I know [bluetooth.c](https://github.com/zephyrproject-rtos/zephyr/blob/2cceb6220642928e08657e63736fab6cd5e0a6b7/subsys/usb/class/bluetooth.c#L92) has something similar, but is making this configurable helps anywhere?

Reply:
Right, there are other config option to setup the ACL length so I don't think we need yet another config option.